docs/endpoints/posts.md

```diff
@@ -168,12 +168,14 @@ This will automatically create or update the corresponding signup.
 POST /api/v3/posts
 ```
 
-- **campaign_id**: (int) required.
+- **campaign_id**: (int) required without action_id.
   The Campaign ID of the campaign that the user's post is associated with.
 - **type**: (string) required.
   The type of post submitted. Must be one of the following types: `photo`, `voter-reg`, `text`, `share-social`. `share-social` posts will be auto-accepted unless an admin sets a custom `status`.
-- **action**: (string) required.
+- **action**: (string) required without action_id.
```

Comment:
I think it might be helpful to "remove" these from the main argument list, and list them below (so we start by listing required and optional fields, and then show these deprecated options beneath). That way there's a bit less "parsing" necessary when skimming the docs.

Reply:
@DFurnes yeah, I agree that it is a little hard to read! Do you mean separate into different lists of required and optional params?

Reply:
Yeah, I think grouping these into "required", "optional", and "deprecated" buckets is a nice idea.

Reply:
great! Added the "required" and "optional" sections for now ("deprecated" to come later when there is anything relevant for this category!).

Reply:
Oh sorry, I wasn't clear! I was suggesting moving `campaign_id` and `action` into the "deprecated" section, since right now it's somewhat hard to parse out that it's "either provide these two fields or this one" and that `action_id` is preferred.

Reply:
OHH I see what you mean! Totally a lot clearer that way - updated!